jsk_fetch_robot/jsk_fetch_startup/config/install_upstart.sh

```diff
@@ -34,3 +35,23 @@ EOF"
 echo "== cat /etc/init/jsk-fetch-startup.conf =="
 cat /etc/init/jsk-fetch-startup.conf
 echo "== usage: sudo service jsk-fetch-startup restart =="
+
+# vertical-touchscreen
+sudo bash -c "cat <<EOF > /etc/init/vertical-touchscreen.conf
+# See https://askubuntu.com/questions/507496/how-to-start-gui-application-with-upstart
+description \"Use vertical touchscreen\"
+
+start on runlevel [2345]
+stop on runlevel [016]
+
+script
+    sleep 30 # Wait for X programs to start
+    export DISPLAY=:0
+    exec su fetch -c \". ${current_prefix_path}/setup.bash && ${jsk_fetch_startup}/scripts/vertical-touchscreen.sh\"
```

Comment:
it looks that it does not depend on ROS, but sould we need to source setup.bash?

Reply:
Thank you for your commend.
I remove unnecessary source.